Base/res/js/Spreadsheet/runtime.js

```diff
@@ -431,38 +431,45 @@ function numericReduce(op, accumulator, cells) {
 }
 
 function numericResolve(cells) {
-    return resolve(cells).map(val => parseInt(val));
+    return resolve(cells).map(val => parseFloat(val));
```

Comment:
This seems to be a pretty significant change. Do we know why the original code was expecting all of these values to be able to be converted to an integer, and why it's ok to accept floating point values for all the callers?

Reply:
I don't think this was a conscious decision, but probably a bug / oversight in the original code. There were no tests that checked or depended on this behavior, and even simple functions like `sum` and `min` gave the wrong results if any inputs were floats.

Reply:
(please split this patch to a separate commit)